.github/workflows/testing.yml

```diff
@@ -24,14 +24,15 @@ jobs:
           version: v1.42.1
 
       - name: Run General Linting
-        uses: docker://ghcr.io/github/super-linter:slim-v4
+        uses: docker://ghcr.io/github/super-linter:slim-v4.8.5
```

Comment:
in theory this should be the same images - https://github.com/github/super-linter/pkgs/container/super-linter
<img width="369" alt="image" src="https://user-images.githubusercontent.com/173697/147750222-0882d24d-91af-4fd7-9e98-5c79e852a989.png">

But I'm fine with pinning the full version as it can be some sort of caching preventing it from using really this latest version